openrest4j-api/src/main/java/com/openrest/olo/dispatches/PickupDispatch.java

```diff
@@ -13,10 +14,15 @@
     private static final long serialVersionUID = 1L;
 
     /** Default constructor for JSON deserialization. */
-    public PickupDispatch() {}
+    public PickupDispatch() {
+        this.isCurbside = false;
+    }
 
-    public PickupDispatch(Date time, String timeGuarantee, Integer charge, Map<String, String> properties) {
+    public PickupDispatch(Date time, String timeGuarantee, Integer charge, Map<String, String> properties,
+                          String curbsideAdditionalInformation, Boolean isCurbside) {
         super(time, timeGuarantee, charge, properties);
+        this.curbsideAdditionalInformation = curbsideAdditionalInformation;
+        this.isCurbside = (isCurbside != null) ? isCurbside : false;

```

Comment:
simple assign is enough - no need for != null

Reply:
done